influxdb3_write/src/write_buffer/table_buffer.rs

```diff
@@ -231,6 +232,17 @@ impl TableBuffer {
 
         Ok(RecordBatch::try_new(schema, cols)?)
     }
+
+    /// Returns an estimate of the size of this table buffer based on the data and index sizes.
+    #[allow(dead_code)]
```

Comment:
Why is this here? Is this like that because it's going to be used in a later PR?

Reply:
Yeah, the lint was complaining

Reply:
Updated with name change to have `_` prefix to avoid the lint and removed the allow.